editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -2425,6 +2425,7 @@ VisualShaderEditor::VisualShaderEditor() {
 	add_options.push_back(AddOption("Max", "Vector", "Functions", "VisualShaderNodeVectorOp", TTR("Returns the greater of two values."), VisualShaderNodeVectorOp::OP_MAX, VisualShaderNode::PORT_TYPE_VECTOR));
 	add_options.push_back(AddOption("Min", "Vector", "Functions", "VisualShaderNodeVectorOp", TTR("Returns the lesser of two values."), VisualShaderNodeVectorOp::OP_MIN, VisualShaderNode::PORT_TYPE_VECTOR));
 	add_options.push_back(AddOption("Mix", "Vector", "Functions", "VisualShaderNodeVectorInterp", TTR("Linear interpolation between two vectors."), -1, VisualShaderNode::PORT_TYPE_VECTOR));
+	add_options.push_back(AddOption("MixS", "Vector", "Functions", "VisualShaderNodeVectorMixScalar", TTR("Linear interpolation between two vectors using scalar."), -1, VisualShaderNode::PORT_TYPE_VECTOR));
```

Comment:
Is it okay that the name is different here than it is below?

Reply:
As mentioned at top I planned to rename VisualShaderNodeVectorInterp to VisualShaderNodeVectorMix - I dont do it now cuz its break up compatiblity..

Reply:
I can rename it to VisualShaderNodeVectorInterpScalar if its desired.. WDYT @akien-mga ?

Reply:
@Chaosus That seems fine to me. I am referring to "MixS" vs "VectorMixScalar"

Reply:
Ah. "MixS" is better to be the same. There is other functions with the same convention eg. DDx - DDxS or Smoothstep - SmoothstepS